File: lib/src/main/java/de/tum/in/tumcampusapp/models/GCMUpdate.java
package de.tum.in.tumcampusapp.models;

import java.io.Serializable;

/**
 * Used for parsing the GCM json payload for the 'update' type
 */
public class GCMUpdate implements Serializable {

    private static final long serialVersionUID = -4597228673980239217L;
    public int packageVersion;
    public int sdkVersion;
    public String releaseDate;
    public int lowestVersion;

}

File: app/src/main/java/de/tum/in/tumcampusapp/models/gcm/GCMUpdate.kt
package de.tum.`in`.tumcampusapp.models.gcm

import java.io.Serializable

/**
 * Used for parsing the GCM json payload for the 'update' type
 */
data class GCMUpdate(var packageVersion: Int = 0,
                     var sdkVersion: Int = 0,
                     var releaseDate: String = "",
                     var lowestVersion: Int = 0) : Serializable {
    companion object {
        private const val serialVersionUID = -4597228673980239217L
    }
}
